contracts/validator_manager.v.py

```diff
@@ -50,63 +74,58 @@
 # Number of receipts
 num_receipts: int128
 
-# Indexs of empty slots caused by the function `withdraw`
-empty_slots_stack: int128[int128]
-
-# The top index of the stack in empty_slots_stack
-empty_slots_stack_top: int128
-
-# Has the validator deposited before?
-is_validator_deposited: public(bool[address])
-
 # Log the latest period number of the shard
 period_head: public(int128[int128])
 
 
 # Configuration Parameter
 
-# The exact deposit size which you have to deposit to become a validator
-deposit_size: wei_value
+# The fixed-size deposit, denominated in ETH, required for registration
+NOTARY_DEPOSIT: wei_value
+
+NOTARY_LOCKUP_LENGTH: int128
 
 # Number of blocks in one period
-period_length: int128
+PERIOD_LENGTH: int128
 
 # Number of shards
 shard_count: int128
```

Comment:
Is "all capital letters with underscores separating words for constants" a conversion of Vyper? If yes, maybe change `shard_count` to `SHARD_COUNT` too?